docs/index.md

```diff
@@ -89,34 +89,71 @@ use-cases.
 
 
 
-
 ## Application Domains
 
-Native features of CrateDB paired with auxiliary software components provide
-optimal coverage of different application domains.
+Learn how to apply CrateDB's features to optimally cover use-cases in different
+application and topic domains.
 
 ::::{grid} 3
 :padding: 0
 
+:::{grid-item-card} Time Series Data
+:link: timeseries
+:link-type: ref
+:link-alt: Managing Time Series Data with CrateDB
+:padding: 3
+:text-align: center
+:class-card: sd-pt-3
+:class-body: sd-fs-1
+:class-title: sd-fs-5
+
+{material-outlined}`query_stats;1.3em`
+:::
 
-:::{grid-item-card} Analysis and Visualization
-:link: analysis
+
+:::{grid-item-card} Machine Learning
+:link: machine-learning
 :link-type: ref
-:link-alt: Data Analysis and Visualization with CrateDB
+:link-alt: Machine Learning with CrateDB
 :padding: 3
 :text-align: center
 :class-card: sd-pt-3
 :class-body: sd-fs-1
 :class-title: sd-fs-5
 
-{material-outlined}`analytics;1.3em`
+{material-outlined}`model_training;1.3em`
 :::
 
+::::
 
-:::{grid-item-card} Time Series Data
-:link: timeseries
+
+## Integrations
+
+Learn how to use CrateDB with 3rd-party software applications, libraries, and
+frameworks.
+
+::::{grid} 3
+:padding: 0
+
+
+:::{grid-item-card} ETL
+:link: etl
 :link-type: ref
-:link-alt: Managing Time Series Data with CrateDB
+:link-alt: Load and export data into/from CrateDB
+:padding: 3
+:text-align: center
+:class-card: sd-pt-3
+:class-body: sd-fs-1
+:class-title: sd-fs-5
+
+{material-outlined}`transform;1.3em`
+:::
+
+
+:::{grid-item-card} Metrics
+:link: metrics
+:link-type: ref
+:link-alt: CrateDB with metrics collection agents, brokers, and stores
```

Comment:
On [clients-tools » Integrations](https://cratedb.com/docs/crate/clients-tools/en/latest/#id1), the icon here is also used for "System Metrics".